### ISSUE TITLE
Close script tag not allowed

### ISSUE THREAD

**Issue by therevolver** [WordPress/gutenberg#15431]
Sometime in the past couple of weeks the gutenberg editor stopped updating or saving posts with <script></script> in the custom HTML or classic block. I believe it is the amp or yoast premium plugin that might be affecting it but I can't figure out any kind of work around.

Either way, it does not allow a post to save, update or publish when that is in there. It works fine when I go with <script /> but that cuts the post pages at the content. Any suggestions would be greatly appreciated or any ideas on how to fix.

**Comment by nerrad** [comment]
Hi there,

Thanks for reporting your issue.  We need more information before we can assist with reproducing and potentially fixing.  Can you please follow the [bug report template](https://github.com/WordPress/gutenberg/issues/new?template=Bug_report.md) when crafting your issue.  The questions there assist with providing the information needed to help with reproducing and addressing reported issues.

There is not enough information in  your report for this to be actionable so I'm going to close this for now but feel free to open a new issue following the template and with more details.
